### PR TITLE
[WIP] feat(client): desgin new homepage

### PR DESCRIPTION
**Pull request by nihalanisumit**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [X] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [X] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by nihalanisumit** [comment]
Deploy preview: https://deploy-preview-111--lumen-prod.netlify.com/

File: client/src/containers/Category.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+const Category = () => {
+    return (
+        <div>
+            <h1> This is a category page which will list down all the categories with around 5 blogs each</h1>
+        </div>
+    )
+}
+
+export default Category;
```
Comment (piyushagwl007) [inline client/src/containers/Category.jsx:11]: Add new line here

File: client/src/containers/CategoryType.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const CategoryType = ({match}) => {
+    return (
+        <div>
+            <h1>{match.params.type}</h1>
+        </div>
+    )
+}
```
Comment (piyushagwl007) [inline client/src/containers/CategoryType.jsx:9]: pass the type by Default if we are not dealing with any other properties of the match variable.

**Review by piyushagwl007 — APPROVED** [review]
LGTM after changes

File: client/src/containers/CategoryType.jsx (hunk 2 above)
Comment (nihalanisumit) [reply client/src/containers/CategoryType.jsx:9]: I am going to work on category in next PR

File: client/src/containers/Category.jsx (hunk 1 above)
Comment (nihalanisumit) [reply client/src/containers/Category.jsx:11]: I am going to work on category in next PR

**Pull request merged**